lib/internal/modules/esm/assert.js

```diff
@@ -42,7 +42,7 @@ const supportedAssertionTypes = ArrayPrototypeFilter(
   ObjectValues(formatTypeMap),
   (type) => type !== kImplicitAssertType);
 
-
+// eslint-disable-next-line jsdoc/require-returns-check
```

Comment:
Perhaps this one should be fixed rather than disabled. Instead of this comment, line 84 could have a `return` added to it:

```js
      return handleInvalidType(url, importAssertions.type);
```